### ISSUE TITLE
Add stylelint as a new Special

### ISSUE THREAD

**Issue by Josehower** [depcheck/depcheck#791]
### Feature request description

Similarly as eslint special it would be good having depcheck reading and listing missing dependencies in `stylelint.config.cjs` files

### Code snippets (if applicable)

in this example repo, we can see how the dependency stylelint and stylelint-config-upleveled are flagged as unused dependencies

to reproduce:

1. go to the example repo
2. run `pnpm depcheck`

https://stackblitz.com/edit/github-4ky4ug?file=stylelint.config.cjs

```console
~/projects/github-4ky4ug 2s
❯ pnpm depcheck
Unused dependencies
* stylelint
* stylelint-config-upleveled
```

### Any extra info

I have created a [draft PR for this feature](https://github.com/depcheck/depcheck/pull/790) that is actually kind of working but need to be polished, revised and also create a test example.

with the current PR this is the output of the depcheck command

```console
➜  dep-check-stylelint-reproduction git:(main) node /home/josehower/depcheck/bin/depcheck.js 
No depcheck issue
➜  dep-check-stylelint-reproduction git:(main)

```
